build.assets/Makefile

```diff
@@ -328,3 +277,28 @@ release-arm64-fips: buildbox-arm-fips
 	@if [ -z ${VERSION} ]; then echo "VERSION is not set"; exit 1; fi
 	docker run $(DOCKERFLAGS) $(BCCFLAGS) -i $(NOROOT) $(BUILDBOX_ARM_FIPS) \
 		/usr/bin/make -C e release -e ADDFLAGS="$(ADDFLAGS)" OS=$(OS) ARCH=$(ARCH) RUNTIME=$(RUNTIME) FIPS=yes VERSION=$(VERSION) GITTAG=v$(VERSION)
+
+#
+# Run docs tester to detect problems.
+#
+.PHONY: docsbox
+docsbox:
+	docker pull $(DOCSBOX)
+
+.PHONY: test-docs
+test-docs: MAKEFILE_DIR := $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
+test-docs: MAKEFILE_ROOT_DIR := $(shell readlink -f $(MAKEFILE_DIR)/..)
+test-docs: DOCS_VERSION := $(shell grep -E ^VERSION $(MAKEFILE_ROOT_DIR)/Makefile | cut -d= -f2 | cut -d. -f1-2)
+test-docs: DOCS_TEMP := $(shell mktemp -d --tmpdir=$(MAKEFILE_ROOT_DIR))
+test-docs: DOCS_TEMP_DIRNAME := $(shell basename $(DOCS_TEMP))
+test-docs: docsbox
+	@chown -R $(UID):$(GID) $(DOCS_TEMP)
+	cd $(DOCS_TEMP) && \
+		git clone https://github.com/gravitational/next . && \
+		rm -rf content/$(DOCS_VERSION) && \
+		ln -s $(MAKEFILE_ROOT_DIR) content/$(DOCS_VERSION)
```

Comment:
does this need to go into `content/`?
can you run those yarn commands against a repo clone in `/tmp/...`?

Reply:
I wanted to reuse the existing `$(DOCKERFLAGS)` which set `--workdir /go/src/github.com/gravitational/teleport` and various other stuff, so it needed to be mounted into a subdirectory from there.

Reply:
Temp dir logic has now been removed.